plugins/flytekit-kf-pytorch/flytekitplugins/kfpytorch/task.py

```diff
@@ -203,7 +203,22 @@ def get_custom(self, settings: SerializationSettings) -> Dict[str, Any]:
 TaskPlugins.register_pythontask_plugin(PyTorch, PyTorchFunctionTask)
 
 
-def spawn_helper(fn: bytes, raw_output_prefix: str, checkpoint_dest: str, checkpoint_src: str, kwargs) -> Any:
+class ElasticWorkerResult(NamedTuple):
+    """
+    A named tuple representing the result of a torch elastic worker process.
+
+    Attributes:
+        return_value (Any): The value returned by the task function in the worker process.
+        decks (list[flytekit.Deck]): A list of flytekit Deck objects created in the worker process.
+    """
+
+    return_value: Any
+    decks: list[flytekit.Deck]
```

Comment:
```suggestion
    decks: List[flytekit.Deck]
```

CI fails with:

```console
flytekitplugins/kfpytorch/task.py:216: in ElasticWorkerResult
    decks: list[flytekit.Deck]
E   TypeError: 'type' object is not subscriptable
```

Seems to use a python version that doesn't support this yet